app/config/security.yml

```diff
@@ -129,6 +129,17 @@ security:
             - ROLE_APP_ADMIN_MAILJET_EMAIL_LIST
             - ROLE_APP_ADMIN_MAILJET_EMAIL_VIEW
 
+        ROLE_ADMIN_REFERENTS:
+            - ROLE_ALLOWED_TO_SWITCH
+            - ROLE_APP_ADMIN_REFERENT_EXPORT
```

Comment:
When you give all permissions like in this case, you can use only `ROLE_APP_ADMIN_REFERENT_ALL` instead of listing all of them